languages/julia/demos/scripts/run_demo.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --time=00-00:30:00
+#SBATCH --ntasks-per-node=4
+
+# RUN REQUIREMENTS: You must do the following to actual use this script
+#    - add the line
+#           #SBATCH --account=<project_handle>
+#      with an appropriate resource allocation in place of <project_handle>
+#    - create a conda environment named 'py-jl-mpi' with the following
+#      python modules installed
+#        * python=3
+#        * numpy
+#        * mpi4py (installed against the same mpi module that is loaded below)
+#        * julia
+#    - an install of julia with the environment given by ./Project.toml
+#      (relative to this script) instantiated and PyCall installed in
+#      the default julia environment
+# See the README.md in this directory for instructions on setting this up.
+
+module purge
+
+module load conda/2019.10
+## Setup shell to use conda activate
```

Comment:
After talking with Harry he said the following:

"conda activate does not work on eagle because it requires changes to the shell environment that cannot be done properly in the module environment manager.  We ask users not to allow conda to modify the .bashrc scripts as this causes conflicts when the module version changes.  Do not run conda init on eagle, it was not designed with a multi-user environment in mind"

So you shouldn't need lines 24-38 here and if you could change line 42 to 'source activate py-jl-mpi'. Sorry for being a pain. I just know that 'conda activate' has led to issues for users in the past